thunder/core/prims.py

```diff
@@ -3353,6 +3354,28 @@ def scatter_add_meta(a: TensorProxy, /, index: TensorProxy, value: TensorProxy,
 scatter_add = make_prim(PrimIDs.SCATTER_ADD, "scatter_add", meta=scatter_add_meta)
 
 
+def scatter_meta(a: TensorProxy, /, index: TensorProxy, src: TensorProxy | Number, dim: int) -> TensorProxy:
+    utils.check_type(src, (TensorProxy, Number, NumberProxy))
+
+    if isinstance(src, TensorProxy):
+        return scatter_add_meta(a, index, src, dim)
+
+    # Shameless scatter_add_meta reuse when `src` is a scalar
```

Comment:
Could you make this comment more technical descriptive and maybe more neutral, please?